packages/devtools/web/devtools_analytics.js

```diff
@@ -20,9 +20,9 @@ function gtag() {
 // Parse the URI parameters.
 let _gtagsValue = new URLSearchParams(window.location.search).get('gtags');
 let _gtagsReset = _gtagsValue == 'reset';
-let _gtagsEnabled = _gtagsValue == 'enabled';
-// Default is disabled if &gtags= is not specified
-let _gtagsDisabled = _gtagsValue == null || _gtagsValue == 'disabled';
+// Default is enabled if &gtags= is not specified
```

Comment:
Comment that this still means users are prompted about tracking analytics.

Reply:
// By default GA is enabled, if &gtags= is not specified. GA is ONLY collected if user's opt-in was accepted.